Test/SilacTest.cs

```diff
@@ -212,8 +212,11 @@ public static void TestSilacQuantification()
             //test proteins
             string[] output = File.ReadAllLines(TestContext.CurrentContext.TestDirectory + @"/TestSilac/AllQuantifiedProteinGroups.tsv");
             Assert.AreEqual(output.Length, 2);
-            Assert.IsTrue(output[0].Contains("Intensity_silac\tIntensity_silac(K+8.014)")); //test that two files were made
-            Assert.IsTrue(output[1].Contains("1237436.8670764582\t618718.4335382291")); //test the heavy intensity is half that of the light (per the raw file)
+            Assert.IsTrue(output[0].Contains("Intensity_silac\tIntensity_silacPart2\tIntensity_silac(K+8.014)\tIntensity_silacPart2(K+8.014)")); //test that two files were made
+
+            //TODO: the heavy peptide in "silacPart2" seems to be missing even though this spectra file is a copy of "silac"?
+            //Assert.IsTrue(output[1].Contains("875000\t875000\t437500\t437500")); //test the heavy intensity is half that of the light (per the raw file)
+            Assert.IsTrue(output[1].Contains("875000\t875000\t437500")); //test the heavy intensity is half that of the light (per the raw file)
```

Comment:
Is there a reason the last intensity was removed?

Reply:
it appears to be a bug but I don't know how long it's been there. it seems to pre-date this change and the unit test never checked for 4 intensity values so it's unclear how long it's been like this. I added a todo item on line 217. I'll see if I can figure out what's going on in the next couple of days. The PSMs get passed correctly into FlashLFQ but one of the four peaks has a 0 intensity value, which is confusing because the peakfinding worked for an identical .mzML file.

Reply:
fixed in 80d4d1a , a list variable was re-used but was modified in the called method